### PR TITLE
Code spliting

### PR DESCRIPTION
**Pull request by jakubsta**

### CODE REVIEW

File: packages/details/index.tsx [hunk 1]
```diff
@@ -57,25 +54,23 @@ const mapStateToProps = (state: IRootState) => {
 };
 
 const mapDispatchToProps = (dispatch) => ({
```
Comment (michalwarda) [inline packages/details/index.tsx]: I wonder if it would be possible to use `bindActionCreators` here.

File: packages/ducks/widget.ts [hunk 2]
```diff
@@ -0,0 +1,61 @@
+import { actionCreatorFactory, isType } from 'typescript-fsa';
+import { Action } from 'redux';
+
+import { EWidgetSize } from '@linkexchange/types/widget';
+// import { IRootState } from './';
```
Comment (michalwarda) [inline packages/ducks/widget.ts]: Remove unused comments

File: widgets/linkexchange-link/src/Banner/containers/Menu.tsx [hunk 3]
```diff
@@ -0,0 +1,24 @@
+import React, { Component } from 'react';
+// import { connect } from 'react-redux';
```
Comment (michalwarda) [inline widgets/linkexchange-link/src/Banner/containers/Menu.tsx]: Remove comments

File: widgets/linkexchange-link/src/index.tsx [hunk 4]
```diff
@@ -2,15 +2,18 @@ import '@webcomponents/custom-elements';
 
 import { render } from 'react-dom';
 import React from 'react';
-import { Store } from 'redux';
-import { Provider } from 'react-redux';
+// import { Store } from 'redux';
```
Comment (michalwarda) [inline widgets/linkexchange-link/src/index.tsx]: Remove comments

File: widgets/linkexchange-link/index.html [hunk 5]
```diff
@@ -10,7 +10,7 @@
     algorithm="links"
     slots=5
     size="leaderboard"
-    asset="rinkeby"
+    asset="ethereum"
```
Comment (michalwarda) [inline widgets/linkexchange-link/index.html]: 🎉

File: packages/details/index.tsx (hunk 1 above)
Comment (jakubsta) [reply packages/details/index.tsx]: Yes, it was possible, I've just changed it to `bindActionCreators` as you suggested.

**Comment by jakubsta** [comment]
Comments removed, I'll merge it soon.

**Pull request merged**